docs/security.html

```diff
@@ -114,4 +115,42 @@
         </ol></li>
 </ol>
 
-<h3><a id="security_authz">7.4 Authorization and ACLs</a></h3>
\ No newline at end of file
+<h3><a id="security_authz">7.4 Authorization and ACLs</a></h3>
+<h3><a id="zk_authz">7.5 ZooKeeper Authentication</a></h3>
+<h4><a id="zk_authz_new">7.5.1 New clusters</a></h4>
+To enable ZooKeeper authentication on brokers, there are two necessary steps:
+<ol>
+	<li> Create a JAAS login file and set the appropriate system property to point to it as described above</li>
+	<li> Set the configuration property <tt>zookeeper.set.acl</tt> in each broker to true</li>
+</ol>
+
+The metadata stored in ZooKeeper is such that only brokers will be able to modify the corresponding znodes, but znodes are world readable. The rationale behind this decision is that the data stored in ZooKeeper is not sensitive, but inappropriate manipulation of znodes can cause cluster disruption.
+
+<h4><a id="zk_authz_migration">7.5.2 Migrating clusters</a></h4>
+If you have a cluster running an older version of simply with security disabled, and you want to make the cluster secure, then for ZooKeeper authentication, you need to execute the following steps in this order:
```

Comment:
"older version of simply" -> "older version of Zookeeper"? Or "older version of Kafka"?